### ISSUE TITLE
[ISSUE]: bug black screen

### ISSUE THREAD

**Issue by Fa3l** [ps2homebrew/FreeDVDBoot#4]
### Checks

- [X] I have checked existing issues for duplicates and found none

### Describe the issue

When I try to start uLaunchelf from Opl or from boot via modchip it stays on a black screen infinitely. In the old version it didn't have this bug

### Console model

SCPH-90010

**Comment by olokos** [comment]
Same problem but with FreeHD Boot, no modchip SCPH-30003R.
wLe build 76200f51387fcdff8acaeef7c75e1386529a9336

the screen flickers once and then just black screen indefinitely

**Comment by dougcarvalho07** [comment]
Same problem, black screen after trying start from OPL USB or from memory card. 

wLe build 76200f5

SCPH-75001

**Comment by ckmop** [comment]
Black screen after run POPS game on USB disk (XX.*.elf)

wLe build https://github.com/ps2homebrew/wLaunchELF/commit/76200f51387fcdff8acaeef7c75e1386529a9336

**Comment by megabytedesu** [comment]
BLACK SCREEN!! on build 76200f5 :(
SCPH-90010

**Comment by pmsobrado** [comment]
Same here, SCPH-77004, using latest pre-release build.

**Comment by dougcarvalho07** [comment]
The black screen still continue on commit e5a6c35 :(

SCPH-75001

**Comment by dougcarvalho07** [comment]
Started working again on commit bb89871. Thanks 👍 😁

**Comment by VisionR1** [comment]
The boot.elf from 27 October merge with freedvdboot VTS_02_0.IFO play, but the latest boot.elf merge with the same again not working only a flicker and after a black screen, the October boot.elf stop at sector 00069AD0 but the new stop at 00069B30

Console model
SCPH-77004_BIOS_VX_PAL_220

**Comment by AKuHAK** [comment]
@skelletR if you run new wle.elf from usb, does it black screen too?

**Comment by VisionR1** [comment]
> @skelletR if you run new wle.elf from usb, does it black screen too?

No, I run the freedvdboot with October boot.elf merge, after go to USB and run the latest boot.elf and is open all good, only if merge the new with freedvdboot

**Comment by AKuHAK** [comment]
> > @skelletR if you run new wle.elf from usb, does it black screen too?
> 
> No, I run the freedvdboot with October boot.elf merge, after go to USB and run the latest boot.elf and is open all good, only if merge the new with freedvdboot

Then, it is not actually a wle issue, but a FreeDVDBoot issue. Moving there>

**Comment by AKuHAK** [comment]
According to [this](https://github.com/ps2homebrew/FreeDVDBoot#replacing-the-initial-program---slim) you need a specially prepared elf file for injecting.

> so it requires that the ELF you load doesn't overwrite those functions I use (those are around 0x84000 - 0x85fff and 0x250000 - 0x29ffff)

> You can run readelf -l to verify your executable satisfies this requirement. For example, this Tetris homebrew just uses 0x00100000 - 0x0017a940:

**Comment by VisionR1** [comment]
> According to [this](https://github.com/ps2homebrew/FreeDVDBoot#replacing-the-initial-program---slim) you need a specially prepared elf file for injecting.
> 
> > so it requires that the ELF you load doesn't overwrite those functions I use (those are around 0x84000 - 0x85fff and 0x250000 - 0x29ffff)
> 
> > You can run readelf -l to verify your executable satisfies this requirement. For example, this Tetris homebrew just uses 0x00100000 - 0x0017a940:

Yeah, this guide this I use, for that say the October boot.elf stop at sector 00069AD0 but the new stop at 00069B30

**Comment by AKuHAK** [comment]
In the guide is said that not all elf files are accepted. I pointed to necessary elf requirements. This is not wlaunchelf fault, but freedvdboot fault, feedvdboot should be modified to accept specific elf.

**Comment by skelletR** [comment]
> 

Oh, understand, thanks you

**Comment by VisionR1** [comment]
> In the guide is said that not all elf files are accepted. I pointed to necessary elf requirements. This is not wlaunchelf fault, but freedvdboot fault, feedvdboot should be modified to accept specific elf.

Oh, understand thanks you
